test/e2e/lib/build.go

```diff
@@ -43,12 +44,23 @@ type BuildConfig struct {
 }
 
 func MakeBuildOrDie(client *Client, config BuildConfig) {
+	client.T.Helper()
+	so := make([]reconcilertestingv1.CloudBuildSourceOption, 0)
+	so = append(so, reconcilertestingv1.WithCloudBuildSourceSink(config.SinkGVK, config.SinkName))
+	so = append(so, reconcilertestingv1.WithCloudBuildSourceServiceAccount(config.ServiceAccountName))
+	build := reconcilertestingv1.NewCloudBuildSource(config.BuildName, client.Namespace, so...)
+	client.CreateBuildOrFail(build)
+
```

Comment:
Could you add more sleep time here as other sources d? like https://github.com/google/knative-gcp/blob/97341712f97d819dfbe622f7582dace079e251a3/test/e2e/lib/auditlogs.go#L100-L102. It is for reducing flakiness in integration test for WI mode.